### PR TITLE
[fix](mv) Fix  not hit mv problem when table has float type

### PR DESCRIPTION
**Pull request by chenlinzhong**
```
CREATE DATABASE test_db;
use test_db;
CREATE TABLE test_db.table_1 (
  k0 BOOLEAN NOT NULL, 
  k1 TINYINT NOT NULL, 
  k2 SMALLINT NOT NULL, 
  k3 INT NOT NULL, 
  k4 BIGINT NOT NULL, 
  k5 LARGEINT NOT NULL, 
  k6 DECIMALV3(9, 3) NOT NULL, 
  k7 CHAR(5) NOT NULL, 
  k8 DATE NOT NULL, 
  k9 DATETIME NOT NULL, 
  k10 VARCHAR(20) NOT NULL, 
  k11 DOUBLE NOT NULL, 
  k12 FLOAT NOT NULL
) PARTITION BY RANGE(k1) (
  PARTITION p1 
  VALUES 
    LESS THAN ("-10"), 
    PARTITION p2 
  VALUES 
    LESS THAN ("0"), 
    PARTITION p3 
  VALUES 
    LESS THAN ("10"), 
    PARTITION p4 
  VALUES 
    LESS THAN ("20")
) DISTRIBUTED BY HASH(k4, k5) BUCKETS 1;


CREATE MATERIALIZED VIEW table_1_mv AS SELECT k0, max(k8), min(k11), sum(k12), count(k10), hll_union(hll_hash(k7)), bitmap_union(to_bitmap(k2)) from test_db.table_1 group by k0

set experimental_enable_nereids_planner=true;
insert into table_1 values(1,2,3,4,5,6,7,"8","2020-10-01","2020-10-10","10",12,13);

EXPLAIN SELECT k0, max(k8), min(k11), sum(k12), count(k10), hll_union(hll_hash(k7)), bitmap_union(to_bitmap(k2)) from test_db.table_1 group by k0
```
result is wrong 
image

after this pr fix problem

image

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/mv/SelectMaterializedIndexWithAggregate.java
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/mv/SelectMaterializedIndexWithAggregate.java:1]: add test case please

**Review by morrySnow — COMMENTED** [review]
replace screenshot with text block for search engine and git commit message friendly

File: fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/mv/SelectMaterializedIndexWithAggregate.java
Comment (chenlinzhong) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/mv/SelectMaterializedIndexWithAggregate.java:1]: > add test case please

ok, later.

**Comment by starocean999** [comment]
Have you tried this on latest master?  or `set enable_sync_mv_cost_based_rewrite=true; `and try again?

**Comment by chenlinzhong** [comment]
> Have you tried this on latest master? or `set enable_sync_mv_cost_based_rewrite=true; `and try again?

i test master  branch , it has been resolved

**Pull request closed without merging**